readme.md

```diff
@@ -140,6 +140,7 @@ Each of the generator properties (like `name`, `address`, and `lorem`) are calle
     firstNameMale                             // 'Maynard'
     firstNameFemale                           // 'Rachel'
     lastName                                  // 'Zulauf'
+    ssn                                       // '123-45-6789'
```

Comment:
no, please add it to the locale-specific section instead of the general section.